api/database/models.py

```diff
@@ -112,7 +112,6 @@ def __init__(self, name, xp, encounter_req, type, level):
         self.level = level
 
 
-
```

Comment:
Some of these classes are double spaced and some are single spaced, let's come to an agreement on newline conventions